src/nix/meson.build

```diff
@@ -212,18 +212,23 @@ nix_symlinks = [
   'nix-store',
 ]
 
+name_suffix = ''
```

Comment:
Doesn't look like we need this anywhere else?

Reply:
It does seem that way for now. Could be factored out later if needed.